src/tests/java/htsjdk/samtools/fastq/FastqRecordTest.java

```diff
@@ -0,0 +1,136 @@
+package htsjdk.samtools.fastq;
```

Comment:
Really? Hardly any of the test files have licenses...is this a thing we want to start now? (I'm not sure why they don't have a license, but I've noticed it...)